kolibri-electron/src/index.js

```diff
@@ -90,7 +90,12 @@ async function loadKolibriEnv(useKey) {
   env.KOLIBRI_HOME = KOLIBRI_HOME;
   env.PYTHONPATH = KOLIBRI_EXTENSIONS;
   env.KOLIBRI_APPS_BUNDLE_PATH = path.join(__dirname, "apps-bundle", "apps");
-  env.KOLIBRI_PROJECT = METRICS_ID
+
+  const APPXMSIX_PATTERN = 'Program Files\\WindowsApps'
+  if (__dirname.includes(APPXMSIX_PATTERN)) {
+    console.log('loading kolibri env, using APPX/MSIX application');
+    env.KOLIBRI_PROJECT = METRICS_ID
+  }
 
   if (!useKey) {
```

Comment:
Thinking about the current workflow, there's still a possible scenario where the app doesn't have the `KOLIBRI_PROJECT` setted, when the app is run from an Endless Key USB, as standalone, but the user choose to download content from the network.

Maybe we can add a fallback here, something like:

```
if (!env.KOLIBRI_PROJECT) {
  env.KOLIBRI_PROJECT = `${METRICS_ID}-standalone`
}
```

Reply:
ok